app.rb

```diff
@@ -25,129 +29,132 @@ def run_option(selected)
       7 => :add_book,
       8 => :add_music_album,
       9 => :add_game,
-      10 => :save_games
+      10 => :save_data
     }
     send(options[selected])
   end
 
   def list_all_games
-    @games.each do |game|
-      puts "id: #{game.id}"
-      puts "genre: #{game.genre}"
-      puts "author first_name: #{game.author.first_name}"
-      puts "author last_name: #{game.author.last_name}"
-      puts "source: #{game.source}"
-      puts "label:  #{game.label}"
```

Comment:
The Label is not printing well, it is printing the class but it would be better if it printed the title and the color